### PR TITLE
add refwidget to subsite

### PR DESCRIPTION
**Pull request by tschanzt**

### CODE REVIEW

File: ftw/subsite/content/subsitedx.py [hunk 1]
```diff
@@ -60,14 +61,12 @@ class ISubsiteSchema(model.Schema):
...
     )
-
+    widget('language_references', ReferenceBrowserWidget,
+           override=True, selectable=["ftw.subsite.Subsite"])
     language_references = RelationList(
         title=_(u'label_language_references', default=u'Languages'),
         default=[],
         missing_value=[],
-        value_type=RelationChoice(title=u'Related Subsites',
-                                  source=SubsiteLanguagObjPathSourceBinder()),
```
Comment (mbaechtold) [inline ftw/subsite/content/subsitedx.py]: Is the class `SubsiteLanguagObjPathSourceBinder` used elsewhere or could it be removed altogether?

File: sources.cfg [hunk 2]
```diff
@@ -5,3 +5,7 @@ extensions = mr.developer
 auto-checkout =
     ftw.simplelayout
     ftw.mobile
+    ftw.referencewidget
+
+[branches]
+ftw.referencewidget = tt-impl
```
Comment (mbaechtold) [inline sources.cfg]: 🚧

File: sources.cfg [hunk 3]
```diff
@@ -5,3 +5,7 @@ extensions = mr.developer
 auto-checkout =
     ftw.simplelayout
     ftw.mobile
+    ftw.referencewidget
```
Comment (mbaechtold) [inline sources.cfg]: 🚧

**Comment by mbaechtold** [comment]
@Tschanzt, could please remove the installation from source and add an entry to the changelog?

**Comment by Tschanzt** [comment]
@mbaechtold updated

**Comment by mbaechtold** [comment]
@tschanzt, I can successfully select the language subsite but if I remove the linked subsite I get form validation error:

screenshot

Can you confirm this?

**Comment by maethu** [comment]
@tschanzt 👈  (poke)

**Comment by tschanzt** [comment]
@maethu @mbaechtold updated.

**Comment by mbaechtold** [comment]
@tschanzt, could you please have a look at the failing test?

**Comment by tschanzt** [comment]
@maethu @mbaechtold This issue is fixed with ftw.referencewidget 1.2.1

**Comment by mbaechtold** [comment]
Cool, thanks @tschanzt 👍

**Pull request merged**